WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -119,7 +119,19 @@ private async Task<InputRegistrationResponse> RegisterInputCoreAsync(InputRegist
 			alice.SetDeadlineRelativeTo(round.ConnectionConfirmationTimeFrame.Duration);
 			round.Alices.Add(alice);
 
-			CoinVerifier?.TryScheduleVerification(coin, round.InputRegistrationTimeFrame.EndTime, confirmations, oneHop: oneHop, cancellationToken);
+			int currentBlockHeight = 0;
+
+			// We need the try-catch block, otherwise the tests will fail.
```

Comment:
Why?

Reply:
I forgot to remove this ugly hack.

Done in [`72252a3`](https://github.com/zkSNACKs/WalletWasabi/pull/10305/commits/72252a3db3cf0b11bf50776b506e000b16f07b9d)